beacon-chain/node/node.go

```diff
@@ -220,25 +186,124 @@ func New(cliCtx *cli.Context, cancel context.CancelFunc, opts ...Option) (*Beaco
 		beacon.BlobStorage = blobs
 	}
 
+	bfs, err := startModules(cliCtx, beacon, depositAddress)
```

Comment:
I don't think modules is the right name here, these are all core services. Maybe `startCoreServices` or `startBaseServices` or something to that effect

Reply:
Yes I struggled a bit to find a name haha.